src/systems/systems/src/shared/arinc429.rs

```diff
@@ -40,6 +40,35 @@ impl<T: Copy> Arinc429Word<T> {
         self.ssm == SignStatus::NormalOperation
     }
 }
+impl Arinc429Word<f32> {
+    pub fn set_bit(&mut self, bit: u32, value: bool) {
+        self.value =
+            (((self.value as u32) & !(1 << (bit - 1))) | ((value as u32) << (bit - 1))) as f32;
+    }
+
+    pub fn get_bit(&self, bit: u32) -> bool {
+        ((self.value as u32 >> (bit - 1)) & 1) != 0
+    }
+}
+impl From<f64> for Arinc429Word<f32> {
```

Comment:
**thought**: This implementation was probably created in anticipation of future reading of the value from a simvar.  **suggestion**: Add a `Reader` implementation as well (next to the `Writer`).